#!/usr/bin/env python
# DWA Planner configuration

from dynamic_reconfigure.parameter_generator_catkin import ParameterGenerator, double_t, int_t, bool_t
gen = ParameterGenerator()

gen.add("max_linear_velocity", double_t, 0, "The maximum desired linear velocity for the robot in m/s", 1.0, 0, 1.5)
gen.add("max_angular_velocity", double_t, 0, "The maximum desired angular velocity for the robot in rad/s", 0.7, 0, 1.5)

exit(gen.generate("base_local_planner", "base_local_planner", "SpeedLimitManager"))
